### PR TITLE
Celery multiple task progress

### PR DESCRIPTION
**Pull request by p1003**

### CODE REVIEW

**Comment by grzanka** [comment]
I think this needs to be tested together with new branch on UI

**Comment by p1003** [comment]
> I think this needs to be tested together with new branch on UI

There are important changes in the API so it has to be tested together

File: README.md [hunk 1]
```diff
@@ -126,11 +129,11 @@ And for Linux:
 curl -i -X POST -b cookies.txt -H "Content-Type:application/json" -d @path/to/jsonfile "http://localhost:5000/sh/run"
 ```
 
-You can also add parameters after ``?`` sign like this: ``http://localhost:5000/sh/run?<param name>=<param value>``
+You can also add parameters after `?` sign like this: `http://localhost:5000/sh/run?<param name>=<param value>`
```
Comment (grzanka) [inline README.md]: Do we still use `sh/run` ?

Comment (p1003) [reply README.md]: no

**Review by grzanka — APPROVED** [review]
LGTM

**Pull request merged**